app/controllers/placements/schools/placements_controller.rb

```diff
@@ -2,12 +2,12 @@ class Placements::Schools::PlacementsController < Placements::ApplicationControl
   before_action :set_school
   before_action :set_placement, only: %i[show remove destroy preview]
   before_action :set_decorated_placement, only: %i[show remove preview]
+  before_action :current_academic_year, only: :index
```

Comment:
Does this need to be a `before_action`?

```suggestion
```

I'm expecting it should be safe to remove because it's already called by `index` through this call chain:

```
index → placements_scope → academic_year_scope → current_academic_year
```